### ISSUE TITLE
Pending issues with demo

### ISSUE THREAD

**Issue by jiggzson** [jiggzson/nerdamer#247]

**Comment by Happypig375** [comment]
I think a demo label would be better?

**Comment by jiggzson** [comment]
Just plain demo?

**Comment by jiggzson** [comment]
@Happypig375 I'm moving all of your demo and documentation related issues here. I'm closing them but I'll reference them from this issue. Cool?

**Comment by Happypig375** [comment]
So when I need to open an issue about demo or docs, I need to post it as a comment here?

**Comment by jiggzson** [comment]
That would be awesome.

**Comment by Happypig375** [comment]
Here is a list:

Demo
---
- #128 
- #163 
- #204 
- #227 
- #228

Documentation
---
- #140 
- #141 
- #146 
- #151 
- #154 
- #160 (Undocumented)
- #165 
- #167 
- #184 
- #215 
- #245

**Comment by Happypig375** [comment]
http://nerdamer.com/functions/divide.html

Divides 2 polynomials.
\-----------------------------------
Usage: nerdamer("divide(x)")

One argument missing. How can one process two things with only one parameter?

**Comment by Happypig375** [comment]
http://nerdamer.com/functions/Expression.text.html
text() returns reserved names...?

**Comment by Happypig375** [comment]
### The grammar police strikes:
http://nerdamer.com/functions/Expression.text.html
>Pass in the string 'decimals' to ~always~ get *the expression back* as decimals.
Pass in the string 'fractions' to ~always~ get *the expression back* as fractions.

http://nerdamer.com/functions/Expression.sub.html
>Substitutes a given value *with* another given value. **(Note the period)**
The value to substitute *with*.

http://nerdamer.com/functions/Expression.toTeX.html
>*Converts the* expression *to its* LaTeX *representation*. **(Note the period)**

http://nerdamer.com/functions/Expression.evaluate.html
>the expression is *simplified* but the functions aren't called

http://nerdamer.com/functions/Expression.variables.html
>Get*s* a list of ~the~ variables contained within the expression. **(Note the period)**

http://nerdamer.com/functions/Expression.buildFunction.html
>The order of ~the~ parameters is in alphabetical order by default but an *array of arguments* can be provided with the desired order. 
The *array of arguments* with the order in which they are preferred.
change the *order of variables* by passing in an array with the *desired* order

http://nerdamer.com/functions/stdev.html
>Return*s the median* symbolic function **(Should be stdev?)** *if it* cannot be calculated. 

Enough for today.

**Comment by Happypig375** [comment]
A few issues while #284 was being written:
After entering 2222222222222222.1, 'Load output to editor' outputs nothing.
After entering 222222222222222.1, 'Load expression to editor' outputs nothing.
After entering 22222222222222222222.1e0, it shows 222222222222222230000 (also the result when using 0.7.13), but 'Load output to editor' outputs 222222222222222222221. How come that button offer more precision than the library itself?

**Comment by Happypig375** [comment]
#306: Update the demo to 0.7.13
@jiggzson This is important for others to not mistakenly think that the library is missing something implemented in 0.7.13.

**Comment by jiggzson** [comment]
@Happypig375. Done.

**Comment by Happypig375** [comment]
Thanks, but putting solveEquations([x+y=3,3x+4y=3]) into demo results in TypeError: symbol.clone is not a function.

**Comment by Happypig375** [comment]
pfactor(78202389238903801/240831735646702201)
=> ((21336787^2)*(23^2))^(-1)*((2803^2)*(99767^2))
(checks DECIMAL)
=> ​((529)⋅(455258479483369))/​((7856809)⋅(9953454289))

**Comment by jiggzson** [comment]
Sounds a little counter intuitive, doesn't it? By definition a decimal number is a*10^n. Why would 23^2 not be expanded to 529 format if you check decimal box? I'm surprised you haven't said anything about the brackets in pfactor. I mean those things redundant as heck. It's already on my TODO list.

**Comment by Happypig375** [comment]
>By definition a decimal number is a*10^n. Why would 23^2 not be expanded to 529 if you check decimal box?

Why go half way then? Why don't literally return 0.32471795724474595?
It is evaluating what it should not evaluate, it should only convert fractions in the form of a/b, not stuff like a^b.

>I'm surprised you haven't said anything about the brackets in pfactor. 

Good catch! I overlooked this. :+1:

> I mean those things redundant as heck.

:laughing:

>It's already on my TODO list.

Great!

**Comment by jiggzson** [comment]
>Why go half way then?

I don't know if I agree. It's doing the conversion on the factors. If you want the factors stripped then just call evaluate.

 I don't see how this is an issue since you're getting more not less. If you already have the desired output then just leave it as-is.

**Comment by Happypig375** [comment]
@jiggzson Let me rephrase. DECIMAL should only convert numeric fractions to numeric decimals. Switching this on causes the return value of pfactor to lose information about the powers on primes, which is a bug.

**Comment by Happypig375** [comment]
solveEquations([a+b=1,a-b=2])
demo=> [[a,3/2],[b,−1/2]]
There is an extra layer of square brackets not actually present in the result.

**Comment by Happypig375** [comment]
"Try it out" button is not present on home page when viewed on mobile
![Screenshot](https://i.imgur.com/jFD9Vbw.png)

Expand button not aligned with other buttons when screen width is too small
![Screenshot](https://i.imgur.com/sUEXuf8.png)
